os/drivers/audio/alc5658.h

```diff
@@ -119,7 +122,7 @@ struct alc5658_dev_s {
 #ifndef CONFIG_AUDIO_EXCLUDE_BALANCE
 	uint16_t balance;			/* Current balance level (b16) */
 #endif							/* CONFIG_AUDIO_EXCLUDE_BALANCE */
-	uint8_t volume;				/* Current volume level {0..63} */
+	uint16_t volume;				/* Current volume level {0..31} */
```

Comment:
why type has changed? any reason?

Reply:
Yes, first it is a 16bit register (easy to store), second, the volume levels in Nuttx are upto 1000, if we have to stick to it, then 8bits wont be enough.

Reply:
volume level as per new comment is 31 only